apis/discovery/v1alpha1/foreigncluster_methods.go

```diff
@@ -88,5 +88,5 @@ func (fc *ForeignCluster) IsExpired() bool {
 		return true
 	}
 	now := time.Now().Unix()
-	return int64(lu+int(fc.Status.TTL)) < now
+	return int64(lu+fc.Spec.TTL) < now
```

Comment:
Unrelated from this specific line, but unclear to me why this file is in the APIs folder.